src/SoapCore.Tests/RawRequestSoap12Tests.cs

```diff
@@ -46,6 +46,33 @@ public void Soap12PingWithActionInHeader()
 			}
 		}
 
+		[TestMethod]
+		public void Soap12PingWithActionInEnvelopeHeader()
```

Comment:
This test is unrelated but to avoid the admin overhead. I have included it to increase test coverage.